lib/redux/reducers/devices.js

```diff
@@ -25,6 +25,21 @@ const devices = {
     source: {type: 'device', driverId: 'Medtronic'},
     enabled: {mac: true, win: true}
   },
+  medtronic600: {
+    instructions: 'Connect your Bayer Contour Next Link 2.4 to your computer',
+    image: {
+      'src': '/images/bayer_contour_next_link_24.jpg',
+      'height': 52,
+      'width': 143,
+      'alt': 'Bayer Contour Next Link 2.4'
+    },
+    key: 'medtronic600',
+    name: 'Medtronic - 630g, 640g or 670g',
```

Comment:
Good point. It was lower case in certain parts of the website, but all the official documentation shows it in upper case. Will fix.